pupil/config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  AZURE_STORAGE_CONNECTION_STRING: process.env.AZURE_STORAGE_CONNECTION_STRING,
+  GOOGLE_SVC_EMAIL: process.env.GOOGLE_SVC_EMAIL,
+  GOOGLE_SVC_PK: process.env.GOOGLE_SVC_PK,
+  GOOGLE_TRACKING_ID: process.env.GOOGLE_TRACKING_ID,
+  GOOGLE_SVC_DOC_KEY: process.env.GOOGLE_SVC_DOC_KEY,
+  MTC_SERVICE: process.env.MTC_SERVICE,
+  MONGO_CONNECTION_STRING: process.env.MONGO_CONNECTION_STRING,
+  NEW_RELIC_LICENSE_KEY: process.env.NEW_RELIC_LICENSE_KEY,
+  PORT: process.env.PORT,
+  SESSION_SECRET: process.env.SESSION_SECRET,
+  STD_LOG_FILE: process.env.STD_LOG_FILE
+}
```

Comment:
I think config should ensure that all values are set to non-empty strings or issue a warning on dev and an error on prod.  It should not start up if the config is missing on prod.